src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -0,0 +1,145 @@
+    { claimantName: claimantName }) }}
+  </p>
+  <p>
+    {{ t('We’ll contact you to let you know how they respond. They can confirm you’ve paid and the claim is ' +
+    'settled, or they can proceed with it.') }}
+  </p>
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationNOForDefendantClaimDetails(claim) %}
+  <h2 class="heading-medium">{{ t('Your response to the claim') }}</h2>
+
+  <p>
+    {{ t('You have rejected the claim. We’ll send you a directions questionnaire – this is a form which you must '
+    + 'complete to tell us more about the claim.') }}
+  </p>
+
+  <p>
+    {{ downloadResponse(claim, DashboardUrlHelper.getStatusUrl(claim), 'Download your response') }}
+  </p>
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationYESForDefendantClaimDetails(claim) %}
+  {% set claimantName = claim.claimData.claimant.name %}
+  <h2 class="heading-medium">{{ t('Your response to the claim') }}</h2>
+  <p>
+    {{ t('You have rejected the claim. You’ve suggested mediation.') }}
+  </p>
+
+  <p>
+    {{ t('We’ll ask {{ claimantName }} if they agree to take part in mediation.',
+    {claimantName: claimantName}) }}
+  </p>
+
+  <p>
+    {{ t('If they say yes, we’ll arrange a date for mediation. '
+    + 'If they say no we’ll send you a directions questionnaire - '
+    + 'this is a form you complete to tell us more about the claim.') }}
+  </p>
+
+  <p>
+    {{ downloadResponse(claim, DashboardUrlHelper.getStatusUrl(claim), 'Download your response') }}
+  </p>
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationNOForDefendantDashboard(claim) %}
+  {{ t('You’ve rejected the claim.' ) }}
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationYESForDefendantDashboard(claim) %}
+  {{ t('You’ve rejected the claim and suggested mediation. '
+  + 'We’ll ask the claimant if they agree to take part in mediation.') }}
+{% endmacro %}
+
+{% macro claimRejectedAlreadyPaidForDefendantDashboard(claim) %}
+  {% set claimantName = claim.claimData.claimant.name %}
+  {{ t('We’ve emailed {{ claimantName }} telling them when and how you paid the claim.',
+  { claimantName: claimantName }) }}
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationNOForClaimantDashboard(claim) %}
+  {% set defendantName = claim.claimData.defendant.name %}
+  {{ t('{{ defendantName }} has rejected your claim.',
+  { defendantName:  defendantName }) }}
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationYESForClaimantDashboard(claim) %}
+  {% set defendantName = claim.claimData.defendant.name %}
+  {{ t('{{ defendantName }} has rejected the claim. '
+  + 'But they’ve suggested a mediation session to help resolve this dispute.',
+  { defendantName: defendantName }) }}
+{% endmacro %}
+
+{% macro claimRejectedAlreadyPaidForClaimantDashboard(claim) %}
+  {% set defendantName = claim.claimData.defendant.name %}
+  {{ t('{{ defendantName }} believes that they’ve paid the claim in full. ',
+  { defendantName: defendantName }) }}
+{% endmacro %}
+
```

Comment:
useless empty line. One empty line at the end of file is enough :)

Reply:
👍